client/src/views/GraphDetail.vue

```diff
@@ -378,7 +378,7 @@ ul {
 .edge-type-list,
 .node-list {
```

Comment:
`.node-list` will be able to be removed once we implement columns for the nodes column.

Reply:
> > Do you have a general strategy in mind for this? If it's a matter of recomputing "columns" by re-ordering the "rows" of data, I can probably help with that bit.
> 
> That is the strategy I had planned, I just wasn't sure how to implement it. Your help would be great!
> 

Ok I'll come and talk to you about the problem and then figure out how I can help.

> > Agreed. Even tabs may not be the right idea, but perhaps a dropdown instead. As you say, let's worry about that later.
> 
> I added a sample select to the sidebar. What do you think we should label it? @waxlamp

"Visualization".

It's sort of in the wrong spot though--right now it's under the "more ways to visualize" heading, which lumps it in with the "external" vis links that are supposed to be there.